frontend/package.json

```diff
@@ -27,6 +27,7 @@
     "@ngx-pwa/local-storage": "^11.1.0",
     "bootstrap": "^4.5.0",
     "immutable": "^4.0.0-rc.12",
+    "ng-starrating": "^1.0.20",
```

Comment:
Possibly need to replace module. See: https://github.com/riteshgandhi/ng-star-rating/issues/37